pkg/k8splan/watcher.go

```diff
@@ -215,6 +216,21 @@ func (w *watcher) start(ctx context.Context) {
 			} else {
 				maxFailureThreshold = -1
 			}
+
+			var maxRetries int
+			if rawMaxRetries, ok := secret.Data[maxRetriesKey]; ok {
+				// max failure threshold is defined. parse and compare
+				maxFailureThreshold, err = strconv.Atoi(string(rawMaxRetries))
+				if err != nil {
+					maxFailureThreshold = -1
+				} else {
+					logrus.Tracef("[K8s] Parsed max retries value of %d", maxRetries)
```

Comment:
Is this supposed to be the following? I don't see `maxFailureThreshold` anywhere else except the else block.
```suggestion
			var maxRetries int
			if rawMaxRetries, ok := secret.Data[maxRetriesKey]; ok {
				// max retries is defined. parse and compare
				maxRetries, err = strconv.Atoi(string(rawMaxRetries))
				if err != nil {
					maxRetries = -1
				} else {
					logrus.Tracef("[K8s] Parsed max retries value of %d", maxRetries)
```

Reply:
`maxFailures` is actually used on the rancher side. This logic can probably be cleaned up to only parse `max-failures` if `max-retries` is not set